content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,221 @@
+   </td>
+   <td>PN_ or SwissProt ID
+   </td>
+  </tr>
+  <tr>
+   <td>Assay
+   </td>
+   <td>Typology
+   </td>
+   <td>Binding, FRET, SPR, Inhibition, phenotypic cellular
+   </td>
+  </tr>
+  <tr>
+   <td>Result Type
+   </td>
+   <td>Potency/Tox 
+   </td>
+   <td>CC50/IC50/EC50/%
+   </td>
+  </tr>
+  <tr>
+   <td>Unit
+   </td>
+   <td>Result unit
+   </td>
+   <td>Concentration/ratio/SI
+   </td>
+  </tr>
+  <tr>
+   <td>Image
+   </td>
+   <td>
+   </td>
+   <td>
+<ul>
+
+<li>OMETIFF
+
+<li>Matrix Format-Zarr
+</li>
+</ul>
+   </td>
+  </tr>
+</table>
+
+
+
+#### Minimum metadata 
+![](https://i.imgur.com/aU2KYV1.png)
+
+
+Minimum metadata set constitutes metadata items that always need to be supplied with any experimental data. For ChEMBL submission, molecular structures and assay description as depicted in the scheme above are suggested as essential metadata. This is basically a subset of the following [schema](https://www.ebi.ac.uk/chembl/db_schema). In case mutated cell lines and/or mutated target proteins have been used in the assay, desirable metadata should be added in the proper group.
+
+The minimum dataset includes three parts: 
+
+1. Assay and project references (mainly literature link and protocol or summary)
+    1. Project level metadata
+    2. Common sample level metadata such as species, tissue, cell type etc.
+2. Compounds reference with structures
+3. Assay results 
+
+
+### Bioactivity data vocabularies
+
+A set of well-established standards and minimum metadata checklists exist for various aspects of CheMBL formatting:
+
+
+
+* **Chemical information ontology (CHEMINF)** [http://semanticchemistry.github.io/semanticchemistry/ontology/cheminf.owl](http://semanticchemistry.github.io/semanticchemistry/ontology/cheminf.owl) 
+
+    CHEMINF covers information about chemical entities. It includes terms for the descriptors commonly used in cheminformatics software applications and the algorithms which generate them.
+
+* **BioAssay Ontology**(BAO)
+
+    [http://www.bioassayontology.org/bao/bao_complete.owl](http://www.bioassayontology.org/bao/bao_complete.owl) 
+
+
+    The BioAssay Ontology (BAO) describes biological screening assays and their results including high-throughput screening (HTS) data for the purpose of categorizing assays and data analysis. BAO is an extensible, knowledge-based, highly expressive (currently SHOIQ(D)) description of biological assays making use of descriptive logic based features of the Web Ontology Language (OWL). 
+
+* __Ontology of units of Measure (OM)__
+ [http://www.ontology-of-units-of-measure.org/resource/om-2](http://www.ontology-of-units-of-measure.org/resource/om-2) 
+ The OM ontology provides classes, instances, and properties that represent the different concepts used for defining and using measures and units. It includes, for instance, common units such as the SI units meter and kilogram, but also units from other systems of units such as the mile or nautical mile.
```

Comment:
```suggestion
 [http://www.ontology-of-units-of-measure.org/resource/om-2](http://www.ontology-of-units-of-measure.org/resource/om-2) 
 The OM ontology provides classes, instances, and properties that represent the different concepts used for defining and using measures and units. It includes, for instance, common units such as the SI units meter and kilogram, and a wide range of units of significance for the field of Chemistry and related information.
It can be easily mapped to other resources such as [Unit Ontology](link to UO), with tools such as [OXO](TODO:provide link to the OXO recipe)
```